pyfar/classes/audio.py

```diff
@@ -58,6 +58,17 @@ def cshape(self):
         """
         return self._data.shape[:-1]
 
+    @property
+    def cdim(self):
+        """
+        Return channel dimension.
```

Comment:
```suggestion
        Return the channel dimension.
```

Reply:
In ``Coordinates`` it's the same for ``cdim``, i would leave it like that